### PR TITLE
Fixed bug in 2nd test for Access Multi-Dimensional Arrays With Indexe…

### LINKED ISSUE #35665: Bug in 2nd test for Access Multi-Dimensional Arrays With Indexes challenge
I see an issue related to the mal-formed 2nd test of this challenge at least once a day on the forum.  An example is [this thread](https://www.freecodecamp.org/forum/t/bracket-notation-in-accessing-multi-dimensional-arrays-with-indexes/266940).

At some point, someone thought and second condition was needed in the assert statement to catch someone trying to solve the challenge by hard-coding the answer like:
```js
var myData = 8;
```
OR
```js
var myData = 3 + 5;
```

It could just be me, but there really is only one way to solve this challenge:
```js
var myData = myArray[2][1];
```
or some variation of the above with white space or leaving off the semi-colon.

Why would the following `testString` not suffice based on the test `text`?  
```yml
assert(/myData=myArray\[2\]\[1\]/.test(code.replace(/\s/g, ''));
```

I was also thinking maybe the `text` could be modified to the following, to make it even more clear:
```yml
  - text: You should only use bracket notation to read the correct value from <code>myArray</code>.
```

At this point, I am looking for feedback.  Once we decide the best course of action, I will add the `first timers welcome` label as it should literally be a copy/paste of the assert we come up with here.

### PR DESCRIPTION
**Pull request by Artistic18**
…s Challenge

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #35665

### CODE REVIEW

**Comment by RandellDawson** [comment]
@Artistic18 Why did you close the PR?

**Comment by Artistic18** [comment]
Sorry I thought I did something wrong.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.english.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ tests:
   - text: <code>myData</code> should be equal to <code>8</code>.
     testString: assert(myData === 8, '<code>myData</code> should be equal to <code>8</code>.');
   - text: You should be using bracket notation to read the correct value from <code>myArray</code>.
-    testString: assert(/myArray\[2\]\[1\]/g.test(code) && !/myData\s*=\s*(?:.*[-+*/%]|\d)/g.test(code), 'You should be using bracket notation to read the correct value from <code>myArray</code>.');
+    testString: assert(/myData=myArray\[2\]\[1\]/.test(code.replace(/\s/g, 'You should be using bracket notation to read the correct value from <code>myArray</code>.'));
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.english.md]: You did not copy/paste the whole assert as referenced in the issue comment.  Technically, you do not need the assert message argument due to some recent changes in the testing code.
```suggestion
    testString: assert(/myData=myArray\[2\]\[1\]/.test(code.replace(/\s/g, ''));
```

**Comment by RandellDawson** [comment]
@Artistic18 Sorry about that.  I did not include one extra `)` in my comment on the original issue, but I have added it now on your behalf.

**Comment by Artistic18** [comment]
No Problem. Thanks

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Review by moT01 — APPROVED** [review]
Looks good @Artistic18, congratulations on your first contribution to freeCodeCamp! 🎊 There's plenty more to do if you're interested. Thanks and happy coding!

**Pull request merged**